recipe/meta.yaml

```diff
@@ -65,11 +65,13 @@ outputs:
         - ninja
         - git    # [linux]
       files:
+        - test.cpp
         - test.cu
         - CMakeLists.txt
         - run_cmake_cuda_tests.sh  # [linux]
       commands:
         - nvcc --version
+        - nvcc --verbose test.cpp
```

Comment:
I think we want this to be `g++ --verbose test.cpp $CXXFLAGS` so we verify that our env works when compilation is not driven by `nvcc`.

Reply:
```suggestion
        - $CXX --verbose test.cpp $CXXFLAGS
```